spec/lib/repository_pulling_worker_spec.rb

```diff
@@ -1,20 +1,22 @@
 require 'spec_helper'
 
-describe SidetiqWorker do
+describe RepositoryPullingWorker do
   subject! { create :repository_with_empty_remote }
-  before{ Worker.clear }
+  before { RepositoryFetchingWorker.clear }
 
   shared_examples 'perform' do |state, minutes, created_jobs_count|
-    describe "state #{state}, imported " << (minutes ? "#{minutes} minutes ago" : "never before") do
+    context("state #{state}, imported " +
+            (minutes ? "#{minutes} minutes ago" : 'never before')) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.